File: revision/test11/devowel_file.pl
#!/usr/local/bin/perl -w

open F, "<", $ARGV[0] or die "cannot open file";
foreach $line (<F>) {
    $line =~ s/[aeiou]//gi;
    push @lines, $line;
}
close F;

open F2,  ">", $ARGV[0] or die "cannot write to file";
print F2 @lines;
close F2;

File: revision/test11/devowel_file.py
#!/usr/bin/python3.6

import sys, re
f = open(sys.argv[1])
lines = []
file_lines = f.readlines()
for line in file_lines:
    line = re.sub('[aeiouAEIOU]','', line)
    lines.append(line)

f2 = open(sys.argv[1], "w")
for line in lines:
    f2.write(line)
    
